azure-pipelines.yml

```diff
@@ -26,8 +26,8 @@ jobs:
     publishToBar: true
```

Comment:
Can you open an issue to follow-up on converting this file to Arcade, too? In theory, our azure-pipelines.yml file is supposed to use templates from `eng/common/`.